From f2b6f342325824399da3525df182031475b5c4dd Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Thu, 22 Jun 2017 13:56:03 +0300
Subject: [PATCH] Link against libintl if needed

When built against a C library that has locale support, but for which
intl support is provided by an external libintl library, psmisc
forgets to link against this library, even though the configure script
properly checks that.

This patch therefore ensure that we link against libintl when needed,
thanks to the @INTLLIBS@ variable provided by the configure script.

Forward-ported Thomas's patch to psmisc 22.16
from commit f066ed70cd6939838d4057f66798cbc1d972cc73
Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
[baruch: move from Makefile.in to Makefile.am for 23.1]
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 9f61ab4f2f70..8ce284e65124 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -65,6 +65,7 @@ src_fuser_LDADD = @LIBINTL@
 src_killall_SOURCES = src/killall.c src/comm.h src/signals.c src/signals.h src/i18n.h
 src_killall_LDADD = @LIBINTL@ @SELINUX_LIB@
 src_peekfd_SOURCES = src/peekfd.c
+src_peekfd_LDADD = @LIBINTL@
 src_pslog_SOURCES = src/pslog.c
 src_pstree_SOURCES = src/pstree.c src/comm.h src/i18n.h
 src_pstree_LDADD = @LIBINTL@ @TERMCAP_LIB@ @SELINUX_LIB@
-- 
2.11.0

